modules/registry-accessing-metrics.adoc

```diff
@@ -56,44 +91,4 @@ imageregistry_http_request_duration_seconds{method="get",quantile="0.99"} 0.0159
 imageregistry_http_request_duration_seconds_sum{method="get"} 12.260727916000022
 ----
 <1> `<user>` can be arbitrary, but `<secret>` must match the value specified in the
```

Comment:
s/but `<secret>` must match the value specified in the registry configuration./but `<secret>` must use the getting token